source/install/native.rst

```diff
@@ -22,23 +22,28 @@ Install QIIME 2 within a ``conda`` environment
 
 Once you have Miniconda installed, create a ``conda`` environment and install the QIIME 2 Core 2017.6 distribution within the environment. We **highly** recommend creating a *new* environment specifically for the QIIME 2 release being installed, as there are many required dependencies that you may not want added to an existing environment. You can choose whatever name you'd like for the environment. In this example, we'll name the environment ``qiime2-2017.6`` to indicate what QIIME 2 release is installed (i.e. ``2017.6``).
 
-The installation process differs slightly across platforms. Please choose the installation instructions that are appropriate for your platform.
 
-macOS/OS X (64-bit)
-~~~~~~~~~~~~~~~~~~~
+.. raw:: html
+   
+   <div class="tabbed panel panel-default">
```

Comment:
`<div class="tabbed panel panel-default">` -> `<div class="tabbed">`

Semantically this isn't a Bootstrap `panel`.